class.jetpack.php

```diff
@@ -4651,7 +4652,7 @@ public static function build_authorize_url( $redirect = false, $iframe = false )
 				'site_lang'     => get_locale(),
 				'_ui'           => $tracks_identity['_ui'],
 				'_ut'           => $tracks_identity['_ut'],
-				'site_created'  => Jetpack::get_assumed_site_creation_date(),
+				'site_created'  => $connection->get_assumed_site_creation_date(),
```

Comment:
Could we use the existing connection manager so we don't instantiate a new connection manager if there's one already?

```suggestion
				'site_created'  => self::connection()->get_assumed_site_creation_date(),
```